CONTRIBUTING.md

```diff
@@ -30,22 +30,14 @@ Then install dependencies and run the tests:
 git submodule update --init --recursive
 make install-tools
 make test
-#if you wish to build platform binaries locally - the step below is needed.
+# if you wish to build platform binaries locally - the step below is needed.
 make build-ui
 ```
 
 ### Running local build with the UI
 
-The `jaeger-ui` submodule contains the source code for the UI assets (requires Node.js 6+). The assets must be compiled first with `make build-ui`, which runs Node.js build and then packages the assets into a Go file that is `.gitignore`-ed. The packaged assets can be enabled by providing a build tag `ui`, e.g.:
```

Comment:
The make target is great, but why remove the explanation? It's valuable information that now someone needs to dig out by reading source code.

Reply:
This is a valid point, @yurishkuro; how the UI assets are created and used deserves a more detailed explanation as it's not straightforward (at least to me) to understand from just looking at code and Makefiles.

I think though, that this instruction should belong in the Pre-requisites section as an explanation to what `make build-ui` does. So my proposal is (what do you think?):

### Pre-requisites
Then install dependencies and run the tests:

```
git submodule update --init --recursive
make install-tools
make test
```

The `jaeger-ui` submodule contains the source code for the UI assets (requires Node.js 6+). The assets must be compiled first with `make build-ui`, which runs Node.js build and then packages the assets into a Go file that is `.gitignore`-ed:

```
# if you wish to build platform binaries locally - the step below is needed.
make build-ui
```

### Running local build with the UI

The packaged assets (created by the `make build-ui` step above) can be enabled by providing a build tag `ui`. The following command will start Jaeger all-in-one with this `ui` tag.
```
make run-all-in-one
```

Reply:
I prefer UI-related explanations to be together under one section.